pgjdbc/src/main/java/org/postgresql/core/QueryExecutor.java

```diff
@@ -412,7 +412,7 @@ Object createQueryKey(String sql, boolean escapeProcessing, boolean isParameteri
    * <p>Don't use this string for logic, only use it when displaying the server version to the user.
    * Prefer getServerVersionNum() for all logic purposes.</p>
    *
-   * @return the server version string from the server_version guc
+   * @return the server version string from the server_version GUC
```

Comment:
This matches line 406. There wasn't any obvious reason to lowercase it here, but I'm happy to drop.

Reply:
this is fine GUC is more correct